nservicebus/upgrades/3to4.md

```diff
@@ -146,6 +146,7 @@ NServiceBus PowerShell cmdlets have moved to NServiceBus.PowerShell.dll. See [Ma
 
 `configure.JsonSerializer()` has been obsoleted, please use `Configure.Serialization.Json();` instead. More info on supported serializers [here](/nservicebus/serialization/).
 
+The `XmlSerializer` will now automatically escape outgoing messages containing invalid characters. Version 3 endpoints will not be able to process those messages correctly.
```

Comment:
That's what I had initially, but it read like tautology...
Because it sounds odd: v4 endpoints can handle invalid characters, but v3 cannot, so I wanted to emphasize that v3 is at fault here, not v4. Which is why you should upgrade :)